src/services/explorer.ts

```diff
@@ -30,25 +30,22 @@ const getERC721 = async (address: string): Promise<Token[]> => {
   return tokenList.filter((token: Token) => token.type === 'ERC-721');
 };
 
-const getBalance = async (address: string): Promise<string | number> => {
-  return axios
-    .get(`https://zksync2-mainnet-explorer.zksync.io/address/${address}`)
-    .then((res) => {
-      const ethInfo = res.data.info.balances['0x0000000000000000000000000000000000000000'];
-      if (!ethInfo) return 'NaN';
-      const ethBalance = (
-        parseInt(ethInfo.balance, 16) *
-        10 ** -ethInfo.tokenInfo.decimals *
-        ethInfo.tokenInfo.usdPrice
-      ).toFixed(2);
-      return parseInt(ethBalance);
-    })
-    .catch((err) => {
-      console.log(err);
-      return 'NaN';
+const getBalance = async (address: string): Promise<Token[]> => {
+  const response: AxiosResponse = await axios.get(`https://zksync2-mainnet-explorer.zksync.io/address/${address}`);
+  const balances = response.data.info.balances;
+  const tokenBalances: Token[] = [];
+  for (const tokenAddress in balances) {
+    const tokenInfo = balances[tokenAddress].tokenInfo;
+    const balance = parseFloat((parseInt(balances[tokenAddress].balance, 16) * 10 ** -tokenInfo.decimals * tokenInfo.usdPrice).toFixed(2));
```

Comment:
You can remove the parseInt method because it is useless and use toFixed(2) to have rounded value without .00000001